docs/adr/0011-down-size-of-base64-image.md

```diff
@@ -0,0 +1,25 @@
+# 11. down-size-of-base64-image
+
+Date: 2022-07-28
+
+## Status
+
+Accepted
+
+## Context
+
+For large images (several MB or more) converting to base64 will take a long time and exceed the storage limit of the string. 
+Leads to long waits on the main thread, causing a feeling of not being smooth on mobile devices with weak configuration.
+
+## Decision
+
+We can reduce the image `size` and `quality` when displaying.
+- For mobile: 
+  + Use [flutter_image_compress](https://pub.dev/packages/flutter_image_compress) to compress image use native (Objective-C/Kotlin) with faster speed.
+- For web:
+  + Use [image_picker](https://pub.dev/packages/image_picker) to resize image when pick image
```

Comment:
You mean, we will reduce the size before uploading? If yes, IMO it is not a good idea. Please keep the message as plain as the people wrote. We should only reduce the size when rendering only.

Reply:
- IMO It's pretty cool solution for web platform. Because if we proceed to resize it, it will take a long time, moreover, our web has not yet implemented multithreading for it, so it is easy to block the main thread.
- This is an `inline` image in the text, so in my opinion users will not pay too much attention to the quality. Because they can send high quality photos through `attachments`.